### DISCUSSION TITLE
Unable to edit docs of presets-regexManagers

### DISCUSSION THREAD

**Discussion by mueller-ma** [renovatebot/renovate#24312] [Request Help] [answered]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I am trying to get this working for the first time

### Describe the problem

- Go to https://docs.renovatebot.com/presets-regexManagers/
- Click on the pencil icon
- Get redirected to https://github.com/renovatebot/renovate/edit/main/docs/usage/presets-regexManagers.md which is a 404 page

### Have you created a minimal reproduction repository?

I have explained in the description why a minimal reproduction is impossible

**Comment by rarkins** [comment]
This page is auto-generated from https://github.com/renovatebot/renovate/blob/main/lib/config/presets/internal/regex-managers.ts

@HonkingGoose do we have the ability to disable or override the automatic editing link in cases like this?

**Reply by HonkingGoose** [reply, marked as answer]
We can override the link edit URL:

> needs to be overriden by our docs generator like here: https://github.com/renovatebot/renovatebot.github.io/blob/a8b7cebc4d9a9d3f3f364352231ecc301c184b55/src/index.md#L2

One of our contributors created this issue, and I assigned it to `@viceice`, I guess they were/are busy with other things:

- #19309

I don't understand the proposed fix, so I'll have to wait for somebody to fix it for me... 🙃 I don't want to break the docs site with a bad hack-job "fix".
